src/components/table/group-rows.tsx

```diff
@@ -0,0 +1,53 @@
+import { computeRowStyle } from "../utils";
+import { GroupBranche } from "../utils/group";
+import { IRowOptions } from "./row";
+
+export interface GroupRowsProps {
+  rowsOptions?: IRowOptions;
+  rowsStyle?: Omit<React.CSSProperties, "height">;
+  /** cell content replacing the display of the group */
+  CustomGroupCell?: React.ComponentType<any>;
```

Comment:
We can add an interface with className, group, groups, index ?

Reply:
yes the GroupCellProps